api_volontaria/apps/position/views.py

```diff
@@ -0,0 +1,76 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from dry_rest_permissions.generics import DRYPermissions, \
+    DRYPermissionFiltersBase
+from rest_framework import viewsets
+from rest_framework.permissions import IsAdminUser
+
+from api_volontaria.apps.position.models import (
+    Position,
+    Application,
+)
+
+from api_volontaria.apps.position.serializers import (
+    PositionSerializer,
+    ApplicationSerializer,
+)
+
+
+class PositionViewSet(viewsets.ModelViewSet):
+
+    serializer_class = PositionSerializer
+    queryset = Position.objects.all()
+    filter_fields = '__all__'
+    permission_classes = (DRYPermissions,)
+
+    def get_permissions(self):
+        if self.action in ['list', 'retrieve']:
+            permission_classes = []
+        else:
+            permission_classes = [IsAdminUser]
+
+        return [permission() for permission in permission_classes]
```

Comment:
By creating the `get_permissions()` functions you overwrite the `permission_classes` attributes that become useless, but i think that the attributes was better than your function implementation since in your function you directly overwrite all the permissions define with DryRestPermissions in your models to just using `IsAdminUser`.

I think you just want to delete this function and keep the attribute, no ?

Reply:
Agreed. Done